src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/Projects/GetCrossPlatformCompatibility.cs

```diff
@@ -0,0 +1,28 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable disable
```

Comment:
what's this needed for?

Reply:
nvm I see the PR that added this in all the other files in the repo

Reply:
please remove these directives from your files if they are not generating nullable related warnings.  I added globally in the repo since it would take too long to only add to files with the warnings.